SSA/Projects/InstCombine/lean/g2004h02h23hShiftShiftOverflow.lean

```diff
@@ -0,0 +1,79 @@
+import SSA.Projects.InstCombine.lean.g2004h02h23hShiftShiftOverflow_proof
```

Comment:
Why do you call the new directory `lean`? Would `tests/LLVM` make more sense?

Reply:
Do you mean that the files should be stored in `SSA/tests/LLVM/*` or in `SSA/Projects/InstCombine/tests/LLVM/*`?

Reply:
SSA/Projects/InstCombine/tests/LLVM/*